.claude/skills/create-domain-layer/SKILL.md

```
@@ -0,0 +1,51 @@
+---
+name: create-domain-layer
+description: >
+  Whiplash 프로젝트에서 새 도메인 생성, 레이어 추가(Entity/Repository/Service/UseCase/Controller) 시 사용.
+  "도메인 만들어줘", "레이어 추가해줘", "UseCase 만들어줘", "Service 구현해줘", "Controller 추가해줘" 요청 시 반드시 참고.
+---
+
+# 도메인 생성 스킬
+
+## 패키지 구조
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)